db/migrate/20200415010506_remove_hits_from_preferences.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveHitsFromPreferences < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :preferences, :hide_all_hit_counts
+    remove_column :preferences, :hide_private_hit_count
+    remove_column :preferences, :hide_public_hit_count
```

Comment:
Is this table large enough that this migration needs the `pt-online-schema-change` treatment? Or can it stay as is?

(Side note: I don't think this migration is reversible, because `remove_column` can only be reversed if you include the type of the column being removed. Not sure if it needs to be reversible, though.)

Reply:
Let's make it reversible for good practice.

This table should be 2.5m (number of users). We were at ~2m when we ran this migration:

https://github.com/otwcode/otwarchive/blob/b3bdab95dec26a004df9ec450efac72251317eec/db/migrate/20190405191806_add_allow_cocreator_to_preference.rb

and we were fine. So I think we're fine without pt-online-schema-change.

Reply:
It was pretty slow on staging so better safe than sorry: #3805.